src/index.js.flow

```diff
@@ -12,16 +13,20 @@ declare export var MODE: {
 
 declare export var SETTINGS: {
   CURRENT_MODE: typeof MODE.RENDER | typeof MODE.HYDRATE,
+  MANIFEST: Manifest,
 };
 
 declare export var LooselyLazy: {
-  init(mode: typeof SETTINGS.CURRENT_MODE): void,
+  init(options: {
+    mode?: typeof SETTINGS.CURRENT_MODE,
+    manifest?: Manifest,
+  }): void,
 };
 
 declare export default typeof LooselyLazy;
 
 export type LazyComponent<C> = ComponentType<ElementConfig<C>> & {
-  getAssetUrls(manifest: Manifest): string[] | void,
+  getAssetUrls(): string[] | void,
```

Comment:
I don't think this change will work with future plans in SSR. Specifically, we want to eventually move preloads outside of the application render and into the service itself. By removing this parameter, it forces consumers to be in a React render, and the capability we want to introduce will not be possible.

I think if you want to do this, then you also need to expose another function that adds in this capability, i.e.

```
// As is
export type LazyComponent<C> = ComponentType<ElementConfig<C>> & {
  getAssetUrls(manifest: Manifest): string[] | void,
  getAssetUrls(): string[] | void,
};

// Also expose function
export const getAssetUrls = (component, manifest) => ...
```

Reply:
I haven't changed the way it works though.  All we have to do is change:
```js
const urls = forPaint.forEach(comp => comp.getAssetUrls(manifest));
```

to

```js
LooselyLazy.init({ manifest });
const urls = forPaint.forEach(comp => comp.getAssetUrls());
```
Or have I missed something in how we do that in SSR?

Reply:
Yes, the `init` method could be called in the SSR service itself, but it's a roundabout way to achieve the outcome imo. I do also worry that the `init` method would later be incompatible with SSR, and that there may be some gotchas when initialising RLL early in the SSR stream vs the stream in the VM. Though I'm not too sure how the `vm` module works with `node_modules` dependencies tbh.

Reply:
I get your point, but I think consistency from consumer's point has higher value than what internally decide to do. For instance, in future it might be easier to use React `.server.js`  to handle client vs server implementations, in a complete transparent way. And we might add other configurable capabilities in future, so having a unique/required way of setting up the lib might come handy.
I'll add a task to add a test to ensure `init` + `getAssetUrls` work in a jsdom-less env